File: Domain/Sprint.cs
namespace Domain
{
    public class Sprint
    {
        public Guid Id { get; set; }

        public string name { get; set; }

        public string description { get; set; }

        public string description_text { get; set; }

        public string priority { get; set; }

        public DateTime created_at { get; set; }

        public DateTime updated_at { get; set; }

        public DateTime date_start { get; set; }

        public DateTime date_end { get; set; }

        public bool is_active { get; set; }

        public string project_id { get; set; }

        public string status { get; set; }

        public string closing_summary { get; set; }

        public ICollection<SprintIssue> issues { get; set; } = new List<SprintIssue>();
    }
}

File: client-app/src/app/models/sprint.ts
import {Issue} from './issue';

export interface Sprint {
    id: string;
    name: string;
    description: string;
    description_text: string;
    priority: string;
    //original_estimated_duration: string;
    //currently_estimated_duration: string;
    //time_logged: string;
    status: string;
    created_at: string;
    issues: Issue[];
    is_active: boolean;
    date_start: string;
    date_end: string;
    //updated_at: string;
    //reporter_id: string;
    //team_id: string;
    //project_id: string;
    //reviewer_id: string;
    //sprint_id: string;
    //assignees: string[];
}
